sklearn/cluster/tests/test_optics.py

```diff
@@ -0,0 +1,342 @@
+# Authors: Shane Grigsby <refuge@rocktalus.com>
+#          Amy X. Zhang <axz@mit.edu>
+# License: BSD 3 clause
+
+import numpy as np
+
+from sklearn.datasets.samples_generator import make_blobs
+from sklearn.cluster.optics_ import OPTICS
+from sklearn.utils.testing import assert_equal, assert_warns
+from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_raises
+
+from sklearn.cluster.tests.common import generate_clustered_data
+
+
+def test_optics():
```

Comment:
The name of the test is not very informative.
What about `test_correct_number_of_clusters`?

Reply:
Thanks, changed.